SettingsList.py

```diff
@@ -3721,15 +3721,22 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             Begin the game with the selected songs already learnt.
         ''',
     ),
-    Checkbutton(
+    Combobox(
         name           = 'ocarina_songs',
         gui_text       = 'Randomize Ocarina Song Notes',
+        default        = 'off',
+        choices        = {
+            'off': 'Off',
+            'frog': 'Frog Songs Only',
+            'warp': 'Warp Songs Only',
+            'all':  'All Songs',
+        },
         gui_tooltip    = '''\
-                         Will need to memorize a new set of songs.
-                         Can be silly, but difficult. Songs are
-                         generally sensible, and warp songs are
-                         typically more difficult.
-                         ''',
+            Will need to memorize a new set of songs.
+            Can be silly, but difficult. Songs are
```

Comment:
```suggestion
            Can be silly, but difficult. Frog songs are
```

Reply:
That's not what it means, it means all songs are generally sensible, just that warp songs are generally harder but still sensible.

Reply:
Maybe “warp songs are typically more difficult than frog songs”?

Reply:
I dunno, I didn't change the tooltip at all, that's just what it's always been.

Reply:
Sure, but this would be a good opportunity to make it clearer.

Reply:
Yeah, alright, added that change